docs/guides/flowing-cred.md

```diff
@@ -1,5 +1,79 @@
+
+### ☔ Cred Minting
+If community members find one of your contributions valuable, they will **Mint**, or in other words, create, Cred, and flow it to you. This Cred  magically appears out of nowhere and flows in your direction. Cred minting is just like rain, new water falling from the sky into your pond.
+
+Example:<br/>
+When someone Mints Cred by reacting to your Discord post with an emoji...
+
+- It creates a new reaction node in the graph
+- It creates an edge from the person reacting (you), to the reaction node
+- It creates an edge pointing from the reaction node to the post receiving the reaction
+  
+
+### 💡 At a Conceptual Level 
+
+When you react to a message with a SourceCred Emoji <img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' /> in Discord :
+
+  - It creates a new reaction node in the graph, which mints Cred 
+  - It creates an edge from the person reacting (you), to the reaction node
+  - It creates an edge pointing from the reaction node to the message receiving the reaction
+  
+Minting new Cred is also a change for the entire Contribution Graph. It will therefore induce a change on how Cred flows to others so as to keep everything in balance. Again, we need to make sure that water is circulating well with no congestions. 
+
+
+
+  The types of contributions and plugins tracked by the SourceCred algorithm are likely to increase. Currently, your contributions are tracked across the these active plugins in the following ways:
+
+
+| 💬 [Discord] | 🧵 [Discourse] | 🦠 [Github]| 🥇 [Initiatives]
+| -- | -- | -- | -- |
+|Someone gives your post any emoji 👍 💯<img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' />| Someone gives your post/reply a ❤️ | You open a pull request  | You accomplish a specific task approved by the community
+
+ Each community can also change the weights of how much Cred is Minted from each action or reaction. For example, the SourceCred community uses the SourceCred emoji <img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' /> to denote extra value, so it Mints more Cred than other emojis. Or, if a community values GitHub pull requests but uses Discourse for down time chats, they could make it so Discourse posts/replies Mint little Cred and GitHub activity Mints more. 
+
+
+
+### 🔥 Cred Evaporation
+Since Cred can be created from nothing by community members, there must be a balance so that Cred maintains its worth. Our system is called **Cred evaporation**, and is similar to how water evaporates from all water sources. For every node on the Contribution Graph, a fixed proportion of its Cred evaporates away, and that fixed proportion is called alpha. The Cred that evaporates replenishes the source from which Cred is Minted, replenishing steams and creating a closed water cycle.
```

Comment:
```suggestion
Since Cred can be created from nothing by community members, there must be a balance so that Cred maintains its worth. Our system is called **Cred Evaporation**, and is similar to how water evaporates from all water sources. For every node on the Contribution Graph, a fixed proportion of its Cred evaporates away, and that fixed proportion is called alpha. The Cred that evaporates replenishes the source from which Cred is Minted, replenishing steams and creating a closed water cycle.
```